### ISSUE TITLE
[Build/Install] Unable to install GraphViz dot program via yum graphviz-rhel.repo

### ISSUE THREAD

**Issue by GadgetSteve** [ellson/graphviz#259]
Ported Issue from Mantis
Original ID: 2454
Reported By: jpchandru

SEVERITY: BLOCK
Submitted: 2014-05-14 06:36:30

OS: CENTOS

OS BUILD: ALL VERSIONS

PLATFORM: REDHAT LINUX 64BIT

DESCRIPTION
-----------
i tried installing GraphViz today and its failing consecutively due to some broken link:

i stored graphviz-rhel.repo into /etc/yum.repos.d/
Command used: yum install 'graphviz*'

--> Finished Dependency Resolution
Error: Package: graphviz-2.38.0-1.el6.x86_64 (graphviz-stable)
           Requires: libANN.so.1()(64bit)
 You could try using --skip-broken to work around the problem
 You could try running: rpm -Va --nofiles --nodigest

Now i tried yum install 'graphviz*' --skip-broken 
It did installed few packages but skipped a lot and then i tried this:

[root@localhost /]# dot -V
bash: dot: command not found

I dont have dot program installed now which used to get installed previously when i was working last week. 

STEPS TO REPRODUCE
------------------
i tried installing GraphViz today and its failing consecutively due to some broken link:

i stored graphviz-rhel.repo into /etc/yum.repos.d/
Command used: yum install 'graphviz*'

ADDITIONAL INFORMATION
----------------------
This started happenning today after i did yum remove 'graphviz*' inorder to install the latest version because the version that i was using previously wasnt supporting png and gif files.

I would really appreciate if you fix this ASAP because its stopping deliverable in my project and we are in critical stage to do a demo to my client.

**Comment by GadgetSteve** [comment]
Closed as duplicate of #260
